packages/components/src/components/post-popovercontainer/post-popovercontainer.scss

```
@@ -41,6 +41,39 @@
   // Keeps the little arrow visible
   overflow: visible;
 
+  // Safe space overlay styles
+  &[data-safe-space]::after {
+    content: '';
+    position: fixed;
+    inset: 0;
+    z-index: calc(#{commons.$zindex-popover} - 1);
+    pointer-events: auto;
+    background: transparent;
+  }
+
+  &[data-safe-space='triangle']::after {
```

See comment about reflection.

```suggestion
  &[safe-space='triangle']::after {
```